src/lua/effil.lua

```diff
@@ -15,7 +15,8 @@ local api = {
     channel = capi.channel,
     type = capi.type,
     pairs = capi.pairs,
-    ipairs = capi.ipairs
+    ipairs = capi.ipairs,
+    allow_table_upvalue = capi.allow_table_upvalue
```

Comment:
May be renade it to `capture_upvalues`?

Reply:
It's not about upvalues at all, only table upvalues